clients/apps/web/src/components/Settings/OrganizationProfileSettings.tsx

```diff
@@ -117,28 +117,24 @@ const OrganizationSocialLinks = () => {
   }
 
   const handleChange = (index: number, value: string) => {
-    if (!value) return
-
-    // Add protocol if missing
-    if (!value.startsWith('https://')) {
-      value = 'https://' + value
-    } else if (value.startsWith('http://')) {
-      value = value.replace('http://', 'https://')
+    const currentFieldValue = socials[index]?.url
+    console.log('currentFieldValue', currentFieldValue)
+    if (currentFieldValue === '') {
+      value = 'https://'+value
     }
 
-    try {
-      const url = new URL(value)
+    // Infer the platform from the URL
+    let newPlatform: schemas['OrganizationSocialPlatforms'] = 'other'
+    try { const url = new URL(value)
```

Comment:
Can we format the the code? It doesn't look great like this. For the record, we have a prettier configuration available in the repo.